tools/src/main/java/org/apache/rocketmq/tools/command/broker/SendMsgStatusCommand.java

```diff
@@ -85,7 +85,7 @@ public void execute(CommandLine commandLine, Options options, RPCHook rpcHook) t
             for (int i = 0; i < count; i++) {
                 long begin = System.currentTimeMillis();
                 SendResult result = producer.send(buildMessage(brokerName, messageSize));
-                System.out.printf("rt:" + (System.currentTimeMillis() - begin) + "ms, SendResult=%s", result);
+                System.out.printf("rt:%sms, SendResult=%s%n", System.currentTimeMillis() - begin, result);
```

Comment:
It's best to keep the format consistent. rt=%sms

Reply:
done